pom.xml

```diff
@@ -68,7 +68,7 @@
         <!-- Databases -->
         <version.postgresql.driver>42.2.5</version.postgresql.driver>
         <version.mysql.server>5.7</version.mysql.server>
-        <version.mysql.driver>8.0.16</version.mysql.driver>
+        <version.mysql.driver>8.0.17</version.mysql.driver>
```

Comment:
That'd warrant a JIRA issue for the purposes of changelog mention.

Reply:
@gunnarmorling Thanks for spotting, this is test leftover